streamer/src/streamer.rs

```diff
@@ -70,6 +71,12 @@ fn recv_loop(
             if exit.load(Ordering::Relaxed) {
                 return Ok(());
             }
+            if let Some(ref in_vote_only_mode) = in_vote_only_mode {
+                if in_vote_only_mode.load(Ordering::Relaxed) {
+                    sleep(Duration::from_millis(1));
+                    continue;
+                }
+            }
```

Comment:
We might need specificity to not turn off the ingestion for the TPU vote packets. During the last outage when TPU votes stopped landing, gossip votes seemed insufficient to allow the network to catch up.

This means that we could be stuck in vote0only mode forever if we turn off TPU ingestion of votes, while at the same time relying on those votes to land to come out of vote-only mode

Reply:
Nvm, I see this is done in the ` fn new_multi_socket(`

Reply:
It is, the vote packets come in a different port. I pass this as Some() for only non-vote TPU ports. Vote-only TPU port is still open.